app/routes/dashboards/dashboard.js

```diff
@@ -793,6 +817,159 @@ export default Ember.Route.extend({
+                            },
+                            {
+                                parameterName: "institution",
+                                parameterPath: ["query", "bool", "filter", 1, "term", "sources"],
+                            },
+                            {
+                                parameterName: "scholar",
+                                parameterPath: ["query", "bool", "filter", 2, "term", "contributors.exact"],
+                            }
+                        ],
+                        widgetSettings: {
+                            mode: 'search'
+                        }
+                    },
+                    {
+                        // Funder select
+                        widgetType: 'dropdown-widget',
+                        name: 'Funders',
+                        width: 3,
+                        facetDash: "search",
+                        facetDashParameter: "funders",
+                        post_body: {
+                            "aggregations": {
+                                "dropdownList" : {
+                                    "terms": {
+                                        "field": 'funders.exact',
+                                        "size": 100
+                                    }
+                                }
+                            }
+                        },
+                        postBodyParams: [
+                            {
+                                parameterPath: ["query", "bool", "minimum_should_match"],
+                                parameterName: "shouldMatch",
+                                defaultValue: 1
+                            },
+                            {
+                                parameterPath: ["query", "bool", "should"],
+                                defaultValue: (()=>{ return transition.queryParams.all ? ucsd_query : undefined; })()
+                            },
+                            {
+                                parameterName: "sources",
+                                parameterPath: ["query", "bool", "filter", 0, "term", "sources"]
+                            },
+                            {
+                                parameterName: "query",
+                                parameterPath: ["query", "bool", "must", 0, "query_string", "query"],
+                                defaultValue: "*"
+                            },
+                            {
+                                parameterName: "topic",
+                                parameterPath: ["query", "bool", "filter", 0, "term", "tags"]
+                            },
+                            {
+                                parameterName: "institution",
+                                parameterPath: ["query", "bool", "filter", 1, "term", "sources"],
+                            },
+                            {
+                                parameterName: "scholar",
```

Comment:
Same as the widget above, this should be "contributors" and "lists.contributors.id.exact" instead of "contributors.exact"